ibm/mas_devops/playbooks/fullstack-roks.yml

```diff
@@ -87,15 +82,13 @@
     db2wh_instance_name: db2w-shared
     db2wh_load_from_config: "true"
```

Comment:
This should be removed, it's no longer used in the role

Reply:
Fixed.